client/src/planwise/client/projects2/handlers.cljs

```diff
@@ -97,6 +103,7 @@
  in-projects2
  (fn [db [_ current-project]]
    (-> db
+       (reset-state)
```

Comment:
This looks a bit out of place here. Is it intended?

Reply:
The intention was to clear the source-types selection when switching projects. Perhaps it is a bit nicer to select the correct type from the source.